fe/fe-core/src/main/java/org/apache/doris/rewrite/InferFiltersRule.java

```diff
@@ -614,7 +614,10 @@ private void buildNewInPredicate(Expr inExpr,
                 for (Expr conjunct : slotEqSlotExpr) {
                     SlotRef leftSlot = conjunct.getChild(0).unwrapSlotRef();
                     SlotRef rightSlot = conjunct.getChild(1).unwrapSlotRef();
-
+                    if (checkSlot.getDesc().getParent().hasExplicitAlias()
```

Comment:
should also process other infers, such as is not null

Reply:
done